src/App.js

```diff
@@ -4,6 +4,7 @@ import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 
 import './styles/app.scss';
 
+import { ListingSupportContextProvider } from './hooks/listingSupport/listingSupportContextProvider';
```

Comment:
So, what are you asking? For an index.js in hooks/listingsupport to export all the listing support this?  I don't think having this stuff exported from just './hooks' seems right when they are not hooks

Reply:
Should the right place then be a different directory and not `../hooks` or we're okay with it as is.

Reply:
Well, from my stand point this is all for supporting the useListingSupport. I think this stuff should not be separate because it makes it harder to find. The fact that we put all hooks in hooks seems to be a separate issue. So I can see importing all the things from just  'hooks/listingSupport' and not the individual files and that makes sense to me.

TBH, there should be a folder somewhere that has the HOC, useListingSupport, etc in it that all support this HOC wrapper. However, we lose history when we move stuff and it is not that pressing.

Reply:
Added an index and updated the places that made sense.